hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAdmin2.java

```diff
@@ -299,7 +299,8 @@ public void testCloseRegionIfInvalidRegionNameIsPassed() throws Exception {
         if (regionInfo.getRegionNameAsString().contains(name)) {
           info = regionInfo;
           try {
-            ADMIN.unassign(Bytes.toBytes("sample"), true);
+            ADMIN.unassign(Bytes.toBytes(
```

Comment:
Ditto. I know this is not your fault but let's fix it too. Change to use assertThrows.

Reply:
Done.